src/lib.rs

```diff
@@ -514,7 +514,7 @@ another matching engine with fixed memory requirements.
 extern crate aho_corasick;
 extern crate memchr;
 extern crate thread_local;
-#[cfg(test)] extern crate quickcheck;
+#[macro_use] #[cfg(test)] extern crate quickcheck;
```

Comment:
Required to use the `quickcheck!` macro. If there is a reason to avoid pulling quickcheck macros I can refactor to use the `quickcheck` function.